tests/unit/core/test_ExchangeRateManager.py

```diff
@@ -1,38 +0,0 @@
-from lbrynet.metadata import Fee
```

Comment:
Moved this test under lbrynet_daemon folder because that's where ExchangeRateManager is